prov/verbs/src/ep_rdm/verbs_ep_rdm.c

```diff
@@ -453,12 +453,25 @@ int fi_ibv_open_rdm_ep(struct fid_domain *domain, struct fi_info *info,
 	_ep->tx_selective_completion = 0;
 	_ep->rx_selective_completion = 0;
 
+	_ep->n_buffs = FI_IBV_RDM_TAGGED_DFLT_BUFFER_NUM;
+	_ep->buff_len = FI_IBV_RDM_DFLT_BUFFER_SIZE;
+	_ep->rndv_threshold = FI_IBV_RDM_DFLT_BUFFERED_SSIZE;
+
 	switch (info->ep_attr->protocol) {
 	case FI_PROTO_IB_RDM:
 		_ep->topcode = IBV_WR_RDMA_WRITE_WITH_IMM;
+		_ep->rq_wr_depth = info->rx_attr->size;
+		_ep->sq_wr_depth = _ep->n_buffs + 1;
 		break;
 	case FI_PROTO_IWARP_RDM:
 		_ep->topcode = IBV_WR_SEND;
+		_ep->rq_wr_depth = info->rx_attr->size;
+		/*
+		 * TODO: More then 1 outgoing send causes hang of bidirectional
+		 * RNDV tests on iWarp devices +1 for eager buffers releasing.
+		 */
+		_ep->sq_wr_depth = 1 + 1;
+		/* _ep->sq_wr_depth = _ep->n_buffs + 1; */
```

Comment:
The first comment explains the reasoning behind _ep->sq_wr_depth = 2. So it would be redundant to have it as 1 + 1. I didn't realize the second comment was original code. Yes, it can be moved under the first one.